src/public/common/app.ts

```diff
@@ -17,10 +17,11 @@ export interface AppDefinition {
  * A ShipEngine Connect app manifest (package.json file)
  */
 export interface AppManifest {
+  id?: string;
   name: string;
   version: string;
   description: string;
   dependencies: Record<string, string>;
   devDependencies: Record<string, string>;
   [key: string]: unknown;
-}
```

Comment:
Why you hate end of file newlines?

Reply:
I don't know why the new line got removed.